### ISSUE TITLE
Cannot read field "invertCrouchMining" 

### ISSUE THREAD

**Issue by darkphan** [PandaDap2006/Pandas-Falling-Trees#62]
Exception caught during firing event: Cannot read field "invertCrouchMining" because the return value of "me.pandamods.fallingtrees.config.FallingTreesConfig.getClientConfig(net.minecraft.world.entity.player.Player)" is null

Whenever this occurs players are not able to get drops until the server is restarted.

Minecraft version
1.20.1

Forge version
47.2.23

Mod version
0.11

Excerpt from log file, as it does not fully crash the server: https://gist.github.com/darkphan/6b53c1dbf9befbb6f36410bbc9469793

**Comment by JiveyGuy** [comment]
Having in 1.20.4: Log

```
23.04 05:41:34 [Server] Server thread/ERROR Failed to handle packet net.minecraft.class_2846@9a84d7a, suppressing error
23.04 05:41:34 [Server] INFO java.lang.NullPointerException: Cannot read field "invertCrouchMining" because the return value of "me.pandamods.fallingtrees.config.FallingTreesConfig.getClientConfig(net.minecraft.class_1657)" is null
23.04 05:41:34 [Server] INFO at me.pandamods.fallingtrees.trees.StandardTree.allowedToFall(StandardTree.java:123) ~[fallingtrees-0.11-fabric-1.20.jar:?]
23.04 05:41:34 [Server] INFO at me.pandamods.fallingtrees.event.EventHandler.makeTreeFall(EventHandler.java:50) ~[fallingtrees-0.11-fabric-1.20.jar:?]
23.04 05:41:34 [Server] INFO at me.pandamods.fallingtrees.event.EventHandler.lambda$makeTreeFall$0(EventHandler.java:42) ~[fallingtrees-0.11-fabric-1.20.jar:?]
23.04 05:41:34 [Server] INFO at java.util.Optional.filter(Optional.java:218) ~[?:?]
23.04 05:41:34 [Server] INFO at me.pandamods.fallingtrees.event.EventHandler.makeTreeFall(EventHandler.java:42) ~[fallingtrees-0.11-fabric-1.20.jar:?]
23.04 05:41:34 [Server] INFO at me.pandamods.fallingtrees.event.EventHandler.onBlockBreak(EventHandler.java:34) ~[fallingtrees-0.11-fabric-1.20.jar:?]
```

**Comment by PandaDap2006** [comment]
@JiveyGuy This has been fixed in the latest version "0.12"
